java/src/jmri/jmrix/openlcb/swing/OlcbSignalMastAddPane.java

```diff
@@ -106,54 +131,156 @@ public OlcbSignalMastAddPane() {
         return Bundle.getMessage("OlcbSignalMastPane");
     }
 
-
     final JCheckBox allowUnLit = new JCheckBox();
 
-    LinkedHashMap<String, JCheckBox> disabledAspects = new LinkedHashMap<>(NOTIONAL_ASPECT_COUNT);
-    final LinkedHashMap<String, EventIdTextField> aspectEventIDs = new LinkedHashMap<>(NOTIONAL_ASPECT_COUNT);
-    final JPanel disabledAspectsPanel = new JPanel();
+    // This used to be called "disabledAspects", but that's misleading: it's actually a map of
+    // ALL aspects' "disabled" checkboxes, regardless of their enabled/disabled state.
+    LinkedHashMap<String, JCheckBox> allAspectsCheckBoxes = new LinkedHashMap<String, JCheckBox>(NOTIONAL_ASPECT_COUNT);
+    final LinkedHashMap<String, EventIdTextField> aspectEventIDs = new LinkedHashMap<String, EventIdTextField>(NOTIONAL_ASPECT_COUNT);
```

Comment:
Why have you replaced `<>` with `<String, JCheckBox>` on the right side? It's not needed and the code gets cleaner and easier to read without it.